pkg/infra/main.go

```diff
@@ -107,6 +112,7 @@ func gameLoop(globalState state.State, agentMap map[commons.ID]agent.Agent, game
 		globalState = new_global_state
 
 	}
+	logging.Log.Infof("Congratulations, The Peasents have escaped the pit with %d remaining.", len(agentMap))
```

Comment:
This form of logging won't work anymore because of https://github.com/SOMAS2022/SOMAS2022/pull/53.
Needs to be replaced with:
```suggestion
	logging.Log(logging.Info, nil, fmt.Sprintf("Congratulations, The Peasents have escaped the pit with %d remaining.", len(agentMap))
```